functions/graphing_functions.py

```diff
@@ -130,6 +130,14 @@ def summarize_params():
             summary[param][mode]["max"] = summary[param][mode]["mean"] + 3.0 * summary[param][mode]["std"]
             summary[param][mode]["step"] = (summary[param][mode]["max"] - summary[param][mode]["min"]) / 10
             summary[param][mode]["value"] = summary[param][mode]["mean"]
+    #Add VPHPL
+    summary['vphpl']={}
+    for mode in params[param]:
+        summary['vphpl'][mode] = {"mean": np.mean(results['vphpl'][mode]), "std": np.std(results['vphpl'][mode])}
+        summary['vphpl'][mode]["min"] = max(summary['vphpl'][mode]["mean"] - 3.0 * summary['vphpl'][mode]["std"], 0)
+        summary['vphpl'][mode]["max"] = summary['vphpl'][mode]["mean"] + 3.0 * summary['vphpl'][mode]["std"]
+        summary['vphpl'][mode]["step"] = (summary['vphpl'][mode]["max"] - summary['vphpl'][mode]["min"]) / 10
+        summary['vphpl'][mode]["value"] = summary['vphpl'][mode]["mean"]
```

Comment:
this duplication is unnecessary and repetitive. Can't you just put vphpl also into the params?
something like 

```
params = generate_all()
results = vphpl_pphpl_calc(params)
# copying over the vphpl to the parameters since it is a param for the pphpl calculation
params['vphpl'] = results['vphpl']
summary = summarize_params()
```